Assets/BossRoom/Scripts/Editor/BakingMenu.cs

```diff
@@ -0,0 +1,148 @@
+        
+        Debug.Log(bakedLights.Length + " Environment lights set to Realtime");
+    }
+
+    [MenuItem("Boss Room/Lighting Setup/Light Probes/Enabled")]
+    static void LightProbesEnabled()
+    {
+        GameObject[] lightProbes = GameObject.FindGameObjectsWithTag("LightingProbes");
+
+        if (lightProbes == null || lightProbes.Length == 0)
+        {
+            Debug.Log("No light probes found in scene");
+        }
+
+        else
+        {
+            foreach (var index in lightProbes)
+            {
+                index.GetComponent<LightProbeGroup>().enabled = true;
+                Debug.Log("Light probes enabled");
+            }
+        }
+    }
+    
+    [MenuItem("Boss Room/Lighting Setup/Light Probes/Disabled")]
+    static void LightProbesDisabled()
+    {
+        GameObject[] lightProbes = GameObject.FindGameObjectsWithTag("LightingProbes");
+
+        if (lightProbes == null || lightProbes.Length == 0)
+        {
+            Debug.Log("No light probes found in scene");
+        }
+
+        else
+        {
+            foreach (var index in lightProbes)
+            {
+                index.GetComponent<LightProbeGroup>().enabled = false;
+                Debug.Log("Light probes disabled");
+            }
+        }
+    }
+    
+    [MenuItem("Boss Room/Lighting Setup/Reflection Probes/Baked")]
+    static void ReflectionProbesBaked()
+    {
+        GameObject[] reflectionProbes = GameObject.FindGameObjectsWithTag("LightingReflectionProbe");
+
+        if (reflectionProbes == null || reflectionProbes.Length == 0)
+        {
+            Debug.Log("No reflection probes found in scene");
+        }
+
+        else
+        {
+            foreach (var index in reflectionProbes)
+            {
+                index.GetComponent<ReflectionProbe>().mode = ReflectionProbeMode.Baked;
+                Debug.Log("Reflection probes set to Baked");
+            }  
+        }
+    }
+    
+    [MenuItem("Boss Room/Lighting Setup/Reflection Probes/Realtime")]
+    static void ReflectionProbesRealtime()
+    {
+        GameObject[] reflectionProbes = GameObject.FindGameObjectsWithTag("LightingReflectionProbe");
+        
+        if (reflectionProbes == null || reflectionProbes.Length == 0)
+        {
+            Debug.Log("No reflection probes found in scene");
+        }
+
+        else
+        {
+            foreach (var index in reflectionProbes)
+            {
+                index.GetComponent<ReflectionProbe>().mode = ReflectionProbeMode.Realtime;
+                Debug.Log("Reflection probes set to Realtime");
+            }  
+        }
+    }
+    
+    [MenuItem("Boss Room/Lighting Setup/All Baked")]
+    static void AllBaked()
+    {
+        EnvironmentLightsBaked();
+        ReflectionProbesBaked();
+        Debug.Log("All lights and reflection probes in scene set to Baked");
+    }
+    
+    [MenuItem("Boss Room/Lighting Setup/All Realtime (except area lights)")]
+    static void AllRealtime()
```

Comment:
Just nitpicky on method names, but seems like all but the `ClearBakedData` method could use a "Set" prefix. 
```suggestion
    static void SetAllRealtime()
```